localrc-template

```diff
@@ -1,7 +1,6 @@
-export VAGRANT_BOX=jammy  # precise, xenial, bionic, etc.
-# for libvirt, you need to use a libvirt box
-#export VAGRANT_BOX=jammy-libvirt
-#export LIBVIRT_DEFAULT_URI="qemu:///system"
+# Any vagrantup box alias *should* work, but who knows
+#export VAGRANT_BOX=...
```

Comment:
I think there'd still be some value in enumerating some known-supported boxes here.

Reply:
yes I've been thinking about that - I don't want it HERE - but somewhere in the project.  I think the README should help steer new comers to their respective provider README and the maintainers of that provider should have their own list of tesed/supported/recommeneded boxes.

We could theoretically make getting that squared a blocker for this change.  We don't currently have a README-virtualbox.md and the existing README-libvirt.md has issues https://github.com/NVIDIA/vagrant-swift-all-in-one/issues/160